consensus/consensus-types/src/safety_data.rs

```diff
@@ -18,7 +18,7 @@ pub struct SafetyData {
     pub one_chain_round: u64,
     pub last_vote: Option<Vote>,
     pub last_order_vote: Option<OrderVote>,
-    pub last_order_voted_round: Option<u64>,
+    pub highest_timeout_round: Option<u64>,
```

Comment:
I think `highest_timeout_round` should be similar to `last_voted_round`, so use u64 and initialized as 0?

Reply:
I initially tried to use u64. But it seems to give me some backward compatibility issues. At some point, it's reading SafetyData from storage and it gave error that it's unable to find the highest_timeout_round value.

Reply:
I "think" the data is only serialized with json, so we can just use #[serde(default)]

Reply:
Using #[serde(default)] now.